src/server/pps/server/api_server.go

```diff
@@ -2001,14 +2011,21 @@ func job(kubeClient *kube.Client, jobInfo *persist.JobInfo, jobShimImage string,
 					Labels: labels,
 				},
 				Spec: api.PodSpec{
+					InitContainers: []api.Container{
+						{
+							Name:            "init",
+							Image:           jobShimImage,
+							Command:         []string{"cp", "/job-shim", "/pfs/job-shim"},
```

Comment:
I strikes me as a little weird that you're putting the job-shim under a directory called `/pfs` given that it really has more to do with pps.